### DISCUSSION TITLE
rows menu is hidden when sticky first column is enabled

### DISCUSSION THREAD

**Discussion by imed-ghomari** [RafaelGB/obsidian-db-folder#354] [Polls]
**Describe the bug**
When scrolling vertically, the menu to the left of the rows is hidden by the first column.

**Expected behavior**
The menu should be visible if the option `sticky first column` is enabled in settings

**Screenshots**

https://user-images.githubusercontent.com/38974541/189478462-ece56d87-915d-49d7-85a4-b50476a677fa.mov

**Desktop (please complete the following information):**
- macOS 12.3.1
- Obsidian 0.15.9

**Comment by RafaelGB** [comment]
It is a design decision to give importance to the information in that column

**Reply by imed-ghomari** [reply]
I was thinking both (the column and the dots) could be visible when the option `sticky first column` is enabled.

**Reply by RafaelGB** [reply]
It should be another custom toggle config

**Reply by imed-ghomari** [reply]
sure, if you think it's necessary
